tests/test_smab.py

```
@@ -650,6 +650,187 @@ def test_smab_predict_raise_when_all_actions_forbidden():
+    region = Segment(intervals=((lo, hi),)).forbidden_region_outside()
+    point = np.array([x])
+    if lo <= x <= hi:
+        assert region(point) <= 0
+    else:
+        assert region(point) > 0
+
+
+@pytest.mark.parametrize(
+    "x0, x1, expected_forbidden",
+    [
+        (0.2, 0.7, False),  # inside [0, 0.5) × [0.5, 1.0] => allowed
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)